socorro/app/socorro_app.py

```diff
@@ -362,6 +362,61 @@ def setup_logger(config, local_unused, args_unused):
     return wrapped_logger
 
 
+def setup_metrics(config, local_unused, args_unused):
+    """Sets up metrics client which is either a DogStatsd client or a LoggingClient
+
+    NOTE(willkg): This is a little gross, but that's mostly to keep all the code in one place and
+    minimal which makes it easier to deal with for now. If this ever grows, then we'll probably want
+    to split this out into its own module.
+
+    This gets tossed in config (along with everything else), so you can use it like this::
+
+        class SomeComponent(RequiredConfig):
+            def something(self):
+                self.config.metrics.increment('somekey')
+
+    """
+    # We do the import here in case there are parts of Socorro that are running in environments that
+    # don't have this library installed.
+    #
+    # FIXME(willkg): We're using a *super* old version of dogstatsd-python which doesn't have the
+    # DogStatsd class. We should upgrade.
+    try:
+        from datadog.dogstatsd import statsd
+    except ImportError as ioe:
+        statsd is None
```

Comment:
I am puzzled by that syntax. As far as I know, it is not valid, as it will always raise a ``NameError`` (since ``statsd`` is not defined here). What is you goal with that? Did you simply want to write ``statsd = None`` instead? 

Also, you declare ``ioe`` but don't use it, maybe you can remove it?